package.json

```diff
@@ -25,6 +25,7 @@
     "stylelint": ">= 16.1.0"
   },
   "dependencies": {
+    "@apostrophecms/require-nested-after-include": "^1.0.0",
```

Comment:
should we rename it to `@apostrophecms/sass-mix-decl` to make it more visible?

Reply:
Whatever you think will be clearest to third party devs, yes. I'm not very familiar with the issue it solves.

Reply:
Actually if we already published this to npm I would leave it alone, not add work.

Reply:
Migrated to `@apostrophecms/stylelint-no-mixed-decls`: https://github.com/apostrophecms/stylelint-no-mixed-decls/pull/1